thoth_user_api/api_v1.py

```diff
@@ -62,12 +62,12 @@ def get_analyze(analysis_id: str):
 
 def get_analyze_log(analysis_id: str):
     """Get image analysis log."""
-    return _get_pod_log(locals(), 'package-extract-', Configuration.THOTH_MIDDLETIER_NAMESPACE)
+    return _get_job_log(locals(), 'package-extract-', Configuration.THOTH_MIDDLETIER_NAMESPACE)
```

Comment:
Should https://github.com/thoth-station/common/blob/master/thoth/common/openshift.py#L299 be called??

Reply:
no... now I see :)

Reply:
It is, in the ``_get_job_log`` wrapper.